src/include/storage/data_table.h

```diff
@@ -138,6 +138,13 @@ class DataTable {
   bool Select(common::ManagedPointer<transaction::TransactionContext> txn, TupleSlot slot,
               ProjectedRow *out_buffer) const;
 
+  enum class VersionChainType {
+    PRE_UPDATE, POST_UPDATE, INSERT, DELETE, VISIBLE, INVISIBLE
```

Comment:
What is PRE_UPDATE and POST_UPDATE, what is VISIBLE and INVISIBLE? Please add some comments to explain

Reply:
Comments added